src/SlackInterface.js

```diff
@@ -4,27 +4,133 @@
 
 /* eslint no-console:0 */
 
-var RtmClient = require('@slack/client').RtmClient;
-var RTM_EVENTS = require('@slack/client').RTM_EVENTS;
+/*******************************************************************************
+ * Globals
+ */
+const RtmClient = require('@slack/client').RtmClient;
+const RTM_EVENTS = require('@slack/client').RTM_EVENTS;
+const AWS = require('./AWS_API.js');
+var rtm;
+
+const token = process.env.SLACK_API_TOKEN || '';
+const DEBUG = process.env.DEBUG || false;
+AWS.DEBUG = DEBUG;
+
+
+/*******************************************************************************
+ * Functions
+ */
 
+var handleRtmMessage = function(message) {
+    if (DEBUG) { console.log('Message:', message) }
+
+    var text = message.text;
+
+    if (keyMessage(text, 'aws ')) {
```

Comment:
Looks good. One issue maybe we should be keeping things like 'aws' as stored variables. Its good practice not to hardcode things like that. Nit picky but maybe we could also have it parse a list. so we could keep things like aws or AWS, amazon web services etc ect.? just a thought.